python/dask_cudf/dask_cudf/tests/test_accessor.py

```diff
@@ -438,3 +438,47 @@ def test_sorting(data, ascending, na_position, ignore_index):
         .reset_index(drop=True)
     )
     assert_eq(expect, got)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        [{"a": 5, "b": 10}, {"a": 3, "b": 7}, {"a": -3, "b": 11}],
+        [{"a": None, "b": 1}, {"a": None, "b": 0}, {"a": -3, "b": None}],
+        [{"a": 1, "b": 2}],
+        [{"b": 3, "c": 4}],
+    ],
+)
+def test_create_struct_series(data):
+    expect = pd.Series(data)
+    ds_got = dgd.from_cudf(Series(data), 2)
+    assert_eq(expect, ds_got.compute())
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        [{"a": 5, "b": 10}, {"a": 3, "b": 7}, {"a": -3, "b": 11}],
+        [{"a": None, "b": 1}, {"a": None, "b": 0}, {"a": -3, "b": None}],
+        [{"a": 1, "b": 2}],
+    ],
+)
+def test_struct_field_a(data):
+    expect = Series(data).struct.field("a")
+    ds_got = dgd.from_cudf(Series(data), 2).struct.field("a")
+    assert_eq(expect, ds_got.compute())
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        [{"a": 5, "b": 10}, {"a": 3, "b": 7}, {"a": -3, "b": 11}],
+        [{"a": None, "b": 1}, {"a": None, "b": 0}, {"a": -3, "b": None}],
+        [{"a": 1, "b": 2}],
+        [{"b": 3, "c": 4}],
+    ],
+)
+def test_struct_field_zero(data):
+    expect = Series(data).struct.field(0)
+    ds_got = dgd.from_cudf(Series(data), 2).struct.field(0)
+    assert_eq(expect, ds_got.compute())
```

Comment:
It would also be nice to check indices other than `0`.  For example, we could call this `test_struct_field_int` and loop over both `0` and `1` for this test.